src/util/data-gov-api.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+interface Response {
+  help: string;
+  success: boolean;
+  result?: {
+    resouce_id: string;
+    fields: {
+      type: string;
+      id: string;
+    }[];
+    records: Record<string, unknown>[];
```

Comment:
with the generic type, you'll be able to use it here:

```suggestion
    records: TRecord[];
```